docs/_guide/styles.md

```diff
@@ -4,66 +4,247 @@ title: Styles
 ```js
-render() {
-  return html`
-    <style>
-      :host([hidden]) { display: none; }
-      :host {
-        display: block; 
-        border: 1px solid black;
+import {LitElement, css} from 'lit-element';
+
+class MyElement extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+    }
+  `;
+}
+```
+
+Multiple values are useful for inheritance:
+
+```js
+class MySubElement extends MyElement {
+  static styles = [
+    super.styles,
+    css`
+      :host([.important]) {
+        color: red;
       }
-    </style>
-    <p>Hello world</p>
+    `
+  ];
+}
+```
+
+Here the subclass is explicitly inheriting the superclass styles, then adding its own.
+
+Static styles are the reccomended way to style LitElements because on browsers that support it, they generate and use a new standard called _[Constuctible Stylesheets](https://wicg.github.io/construct-stylesheets/)_, with a fallback for browsers that don't.
+
+Constuctible Stylesheets allow LitElement to parse styles exactly once and reuse the resulting Stylesheet object for maximum efficiency.
+
+### Limitations
+
+Static styles apply to all instances of an element. Any expressions in the style text are evaluated and included once, then reused for all instances. If you need to vary styles per-instance we reccomend using CSS custom properties, or inline styles.
+
+For security reasons, the only types of values that can be interpolated into static styles are values returned by the `cssLiteral` and `unsafeCSS` template tags.
+
+```js
+const mainColor = cssLiteral`red`;
+
+class MyElement extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+      color: ${mainColor}
+    }
   `;
 }
 ```
 
-{% include project.html folder="docs/style/hostselector" openFile="my-element.js" %}
+## Inline styles
 
-See the MDN documentation on [:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:host), [:host()](https://developer.mozilla.org/en-US/docs/Web/CSS/:host()), and [pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) for more information.
+You can also style a shadow root with inline styles right in your element template. We still reccomend static styles, but in some cases you may want to vary the CSS per-element. One way to do this is with bindings in `<style>` elements. It's important to note that this will not work with shadow DOM polyfills like ShadyCSS.
 
-### Set host element display styles
+```js
+import {LitElement, property} from 'lit-element';
+
+class MyElement extends LitElement {
+  @property() mainColor = 'blue';
+  render() {
+    return html`
+      <style>
+        :host {
+          color: ${this.mainColor};
+        }
+      </style>
+    `;
+  }
+}
+```
 
-Two best practices for working with custom elements are:
+We strongly reccomend static styles, CSS custom properties, or [lit-html's `classMap` or `styleMap` directives](TODO) if you're stying non-host shadow root contents.
```

Comment:
what does non-host shadow root contents mean? could you give an example? I'm not sure we need to cover it